README.md

```diff
@@ -30,14 +30,31 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+Both .forEach and .map call on the function for each array element, but .map does this and returns a new array with its results, . forEach does not.
+
 2. What is the difference between a function and a method?
 
+A method is a function inside an object, i.e when a value of an object is function.
+
 3. What is closure?
 
+A closure is just a function that has values in there that they have returned. it gives us memory, it gives us the ability to execute a program and remember the previous result of the time that it was invoked.
```

Comment:
A closure is a feature in JavaScript where an inner function has access to the outer (enclosing) function's variables but the outer function does not have access to the inner function variables